server/sonar-web/src/main/js/apps/web-api/components/Menu.tsx

```diff
@@ -49,22 +49,16 @@ export default function Menu(props: Props) {
       <div className="list-group">
         {filteredDomains.map(domain => (
           <Link
-            key={domain.path}
             className={classNames('list-group-item', {
               active: isDomainPathActive(domain.path, splat)
             })}
+            key={domain.path}
             to={'/web_api/' + domain.path}>
             <h3 className="list-group-item-heading">
               {domain.path}
               {domain.deprecated && <DeprecatedBadge />}
               {domain.internal && <InternalBadge />}
             </h3>
-            {domain.description && (
```

Comment:
I think that must stay

Reply:
I think we should maybe discuss with Laura and/or Simon then

Reply:
I suggested this move, so I'm +1 to drop obviously. The display in sidebar prevents from having clean and complete descriptions (so basically longer texts). In the current state descriptions are usually duplications of web service names.